lib/topological_inventory/amazon/collector.rb

```diff
@@ -114,6 +122,9 @@ def save_inventory(collections, refresh_state_uuid = nil, refresh_state_part_uui
             :refresh_state_part_uuid => refresh_state_part_uuid,
           )
         )
+      rescue => e
+        logger.error("Error when sending payload to Ingress API. Error message: #{e.message}. Error headers: #{e&.response_headers}")
+        raise e
```

Comment:
You can just `raise` no need to `raise e`, it'll reraise the last exception for you

Reply:
done